flask_caching/backends/rediscache.py

```diff
@@ -164,6 +164,17 @@ def inc(self, key, delta=1):
     def dec(self, key, delta=1):
         return self._write_client.decr(name=self.key_prefix + key, amount=delta)
 
+    def unlink(self, *keys):
+        """when redis-py >= 3.0.0 and redis > 4, support this operation
+        """
+        if not keys:
+            return
+        if self.key_prefix:
+            keys = [self.key_prefix + key for key in keys]
+        unlink = getattr(self._write_client, "unlink", None)
+        if unlink is not None and callable(unlink):
```

Comment:
Is there really a case where `unlink` is not `None` here and is not a callable?

Reply:
I just want to confirm that he does exist.